src/app/dashboard/Missions/MissionEditView.jsx

```diff
@@ -456,6 +452,7 @@ const MissionEditView = ({ mission, toDetailsView, toListView, volunteers }) =>
             </Card>
 
             <Row classes={classes}>
+              {" "}
```

Comment:
space

Reply:
shoot, not sure why these are getting in there.  I'm DEFINITELY not adding on those on my own.